proto/internal/temporal/server/api/archiver/v1/message.proto

```diff
@@ -51,7 +51,7 @@ message HistoryBlob  {
 }
 
 // ArchiveVisibilityRequest is request to Archive single workflow visibility record
-message ArchiveVisibilityRequest {
+message VisibilityBlob {
```

Comment:
Not sure about this name. `VisibilityRow` or `VisibilityRecord` probably better ones.

Reply:
VisibilityRecord